Content.Server/Anomaly/Effects/IceAnomalySystem.cs

```diff
@@ -0,0 +1,154 @@
+using Content.Server.Weapons.Ranged.Systems;
+using Content.Server.Atmos.EntitySystems;
+using Content.Server.Interaction;
+using Content.Shared.Anomaly.Components;
+using Content.Server.Explosion.EntitySystems;
+using Content.Shared.Projectiles;
+using Content.Shared.Anomaly.Effects.Components;
+using Content.Shared.Mobs.Components;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Random;
+
+namespace Content.Server.Anomaly.Effects;
+
+/// <summary>
+/// This handles <see cref="IceAnomalyComponent"/> and the events from <seealso cref="AnomalySystem"/>
+/// </summary>
+public sealed class IceAnomalySystem : EntitySystem
+{
+    [Dependency] private readonly EntityManager _entman = default!;
```

Comment:
```suggestion
```
All entitySystems contain an implicit reference to EntityManager. Just use the system proxy methods.